front/app/components/ParticipationCTABars/ParticipationCTAContent/index.tsx

```diff
@@ -0,0 +1,173 @@
+} from '@citizenlab/cl2-component-library';
+
+// services
+import { IPhaseData } from 'services/phases';
+
+// utils
+import { getPeriodRemainingUntil } from 'utils/dateUtils';
+
+// i18n
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from '../messages';
+
+// styling
+import styled, { useTheme } from 'styled-components';
+import { maxPageWidth } from 'containers/ProjectsShowPage/styles';
+
+const BlickingIcon = styled(Icon)<{ showAnimation: boolean }>`
+  animation-name: blink-animation;
+  animation-duration: 1.8s;
+  animation-duration: ${({ showAnimation }) => (showAnimation ? '1.8s' : '0s')};
+  animation-delay: 1s;
+  animation-timing-function: ease-in-out;
+  animation-iteration-count: 2;
+
+  @keyframes blink-animation {
+    0% {
+      opacity: 1;
+    }
+    50% {
+      opacity: 0;
+    }
+    100% {
+      opacity: 1;
+    }
+  }
+`;
+
+type Props = {
+  timeLeft?: string;
+  hasUserParticipated?: boolean;
+  CTAButton?: React.ReactNode;
+  currentPhase: IPhaseData | null;
+};
+
+export const ParticipationCTAContent = ({
+  currentPhase,
+  CTAButton,
+  hasUserParticipated = false,
+}: Props) => {
+  const theme = useTheme();
+  const isSmallerThanXlPhone = useBreakpoint('phone');
+  const timeLeft = currentPhase
+    ? getPeriodRemainingUntil(currentPhase.attributes.end_at, 'weeks')
+    : '';
+  let message = hasUserParticipated
+    ? messages.userHasParticipated
+    : messages.projectOpenForSubmission;
+
+  if (isSmallerThanXlPhone && !hasUserParticipated) {
+    message = messages.mobileProjectOpenForSubmission;
+  }
+
+  if (isSmallerThanXlPhone) {
+    return (
+      <Box
+        display="flex"
+        alignItems="center"
+        justifyContent="space-between"
+        flexDirection="row"
+        width="100%"
+        bgColor={theme.colors.tenantPrimary}
+        height="62px"
+        p="20px"
+      >
+        <Box
+          display="flex"
+          justifyContent="center"
+          alignItems="center"
+          flexDirection="column"
+        >
+          <Text color="white" m="0px" fontWeight="bold" fontSize="s">
+            <FormattedMessage {...message} />
+          </Text>
+          {timeLeft && (
+            <Text
+              color="white"
+              style={{ textTransform: 'uppercase' }}
+              m="0px"
+              width="100%"
+              fontSize="xs"
+              my="0px"
+            >
+              <FormattedMessage
+                {...messages.participationTimeLeft}
+                values={{
+                  timeLeft: timeLeft,
+                }}
+              />
+            </Text>
+          )}
+        </Box>
+        <Box display="flex" alignItems="center">
+          {CTAButton}
+        </Box>
+      </Box>
+    );
+  }
+
+  return (
+    <Box
+      display="flex"
+      alignItems="center"
+      justifyContent="space-around"
+      flexDirection="row"
+      width="100%"
+      bgColor={theme.colors.tenantPrimary}
+      height="62px"
```

Comment:
Is it possible to have a multiple of 4 here? :)

Reply:
Indeed, we should align with the design team too on this. Some times it is easy to pick direct values from design